packages/app-runtime/src/AppRuntimeErrors.ts

```diff
@@ -1,5 +1,41 @@
 import { UserfriendlyApplicationError } from "./UserfriendlyApplicationError";
 
+class AppStringProcessor {
+    public wrongURL(): UserfriendlyApplicationError {
+        return new UserfriendlyApplicationError("error.appruntime.AppStringProcessor.wrongURL", "The given URL is in a wrong format.");
```

Comment:
```suggestion
        return new UserfriendlyApplicationError("error.appruntime.appStringProcessor.wrongURL", "The given URL is in a wrong format.");
```
Seems to match our naming scheme better, although we are not consistent in this file. Same for the other error codes.

Reply:
I was matching this file. but ok

Reply:
Should we adjust the other capitalized error codes in this file, too, for consistency (doesn't have to be part of this PR)? Or would that lead to required adjustments that are unnecessary?

Reply:
I have absolutely no opinion on that.